articles/how-to-optimize-wordpress-website-speed/index.md

```diff
@@ -0,0 +1,159 @@
+revisions, data that was created by already uninstalled plugins, and WordPress sessions. To optimize the database, you can use plugins. It is advisable to back up your database before trying to clean. 
+
+Some of the plugins available for database optimization are [WP-Optimize](https://wordpress.org/plugins/wp-optimize/) and [WP-Sweep](https://wordpress.org/plugins/wp-sweep/).
+
+![](wp-optimize-database-optimization.jpeg)
+
+See above screenshot of WP-Optimize 3.1.2 free version database optimization
+
+### Use Lazy Loading for Images and Videos
+
+Lazy loading is a web optimization technique where visible content is loaded first while content appearing below the fold is only downloaded and rendering when necessary. Instead of loading every image and video on a webpage,lLazy loading downloads and render only what is visible to the user's screen.
+
+As the user scrolls down the page, more content is loaded. This significantly increases the loading speed of the content that is visible
+to the user. 
+
+You can activate lazy-loading using plugins such as [Lazy Load for Images](https://wordpress.org/plugins/lazy-load-for-images/) and [BJ Lazy Load](https://wordpress.org/plugins/bj-lazy-load/). 
+
+### Limit the Number of Post Revisions
+
+WordPress has a very good feature called post revisions. Whenever you update a page or a post, another version of the post is created and saved in the database. Overtime, post revisions can increase to a point where they can slow down your WordPress database. In the screenshot below, post revisions have accumulated to 93.
+
+![WordPress post revisions screenshot](wordpress-post-revisions-screenshot.jpeg)
+
+The solution for post revisions is to clean them from the database. You can use the WP-Optimize plugin to clean post revisions as shown in the screenshot below.
+
+![WP-Optimize post revisions cleanup](wp-optimize-post-revisions-cleanup.jpeg)
+
+See the above screenshot of WP-Optimize 3.1.2 deleting post revisions
+
+The next step after cleaning post revisions is to limit the number of post revisions saved for each post. This can be done by adding this line of code to the WordPress wp-config.php file. 
+
+Set the post revisions limit to a number of your choice. A maximum of four revisions is recommended. Note that if you limit the revisions to four, you will only be able to access the last for versions of your page or post.
+
+```php
+define('WP_POST_REVISIONS', 4 )
+```
+
+![WordPress wp-config editing](wp-config.php-edit.jpg)
+
+### Update PHP to the Latest Version
+
+PHP is open-source and undergoing active development to improve on performance, fix bugs, enhance security, and other features. Therefore, there is no doubt that the latest PHP version is more enhanced in terms of performance compared to the previous versions. Keep your PHP version updated to get the benefits. 
```

Comment:
Consider giving a brief explanation of what PHP is. Readers new to WordPress may not understand that PHP is a programming language. Also, consider giving a tangible metric on how much better new version of PHP are. For example, the speed increase in moving from PHP 5.6 to PHP 7.